src/utilities/helpers.ts

```diff
@@ -1,5 +1,13 @@
+import AsyncStorage from "@react-native-community/async-storage";
+import moment from "moment";
+
 const SECONDS_IN_AN_HOUR = 3600;
 
+interface Streak {
```

Comment:
(noob question) What is an interface?

Reply:
A Typescript type, JSON.parse() returns an any so it's good to type it properly